### DISCUSSION TITLE
Notification of new programs announcements for already implemented providers

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#130] [General]
### Update 01/22/2024

https://discord.com/channels/906917846754418770/908054304332603402/1198974150476316722
> We're about to change the approach how we deal with feature requests. As you already know, we check for some some existing providers' new dApps on regular basis and we don't reward when you ping us about them (like new Pact farms or new NFT Prize Game app).

We're going to extend that to **Cometa** staking and farming, **Humble** staking and farming, as well as to new **xBacked** vaults.

Related scripts are created and we're going to power them once or multiple times per day. If a new dApp has arrived and script shows that then we'll deploy such a dApp without any issue created.

Your task as community members is to ping us here or in the related monthly GitHub issue we'll be creating every rewards cycle. We won't be rewarding individual pings, rather members' activity in the related cycle. We'd probably merge such a GitHub issue with the existing [Missing icons and NFT collections thread](https://github.com/asastats/channel/issues/931) and the same "activity policy" will apply for new icons and NFT collections too.

-----

Right now we are leaning on the users' request here in our [issues](https://github.com/asastats/channel/issues) or in our community channels **after** those programs are already deployed on Mainnet. For example, our user reported [a Pact token](https://github.com/asastats/channel/issues/114) being placed under NFT section and we have realized that the arrival of Folks Finance's new programs caused that bug.

Here's a list of programs for which we'll need announcements about an update:

- AlgoDAO
  - [staking](https://app.algodao.fi/staking)

- Algofund
  - [staking](https://app.algofund.cc/staking)  
  - [farming](https://app.algofund.cc/farming)

- AlgoRai
  - [new vaults](https://app.algorai.finance/)
  we update new withdrawal apps weekly, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

- Cometa hub
  - we update [new staking programs and farming apps](https://app.cometa.farm) daily, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

- DBD staking
  - [stake](https://stake.daybyday.io/algorand)

- Folks Finance
  - [new deposit assets](https://docs.folks.finance/developer/contracts) 

- Humble DeFi
  - we update [new staking programs and farming apps](https://app.humble.sh) daily, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

- Opulous staking
  - [new pools](https://app.opulous.org/pools/)

- Pact farming
  - we update [new farm apps](https://app.pact.fi/farms) daily, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

- xBacked vaults
  - we update [new vaults](https://app.xbacked.io/) daily, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

- rug.ninja deployments
  - we update Tinyman2 pools daily, ping us if you notice they have deployed on Mainnet and they are not shown in ASA Stats

**Comment by ipaleka** [comment]
by TheShaman in [HumbleDefi Discord](https://discord.com/channels/909895062262005770/909895951815168000/985967144556724314):

- - - - -

🚀Humble V2 is going live June 14th! 1PM CDT 🚀

Swap, Pool, and Earn on your favorite assets and get ready for our Yield Farms powered with 2million #ALGO  and matched ASA funds, launching June 21st. 

We have missed you #algofam🤝

Its going to be a great week @everyone stay tuned to hear the 10+ Farms that we will be rolling out :Launch:

- - - - -

**Comment by ipaleka** [comment]
https://twitter.com/AlgoFundco/status/1536454983851331588

New staking pool available on [http://app.algofund.co](https://t.co/LMCowrbesP)! 

AO x ALGO

**Comment by ipaleka** [comment]
Folks added Tinyman and Pact USDC/USDt LP tokens programs.

**Comment by wxivd** [comment]
**Glitter Finance** $XGLI staking pools announced by [Yieldly on Twitter](https://twitter.com/YieldlyFinance/status/1537865927072546824?s=20&t=kTZ5OnhPC1f8-AytznyU3w), starting next week

**Reply by wxivd** [reply]
by [Yieldly on Twitte](https://twitter.com/YieldlyFinance/status/1539185336630366209?s=20&t=eUPx6fD_sdrhJlgdi1SYyA)

XGLI staking - This week

- YLDY > XGLI
- XGLI/YLDY LP > YLDY

**Reply by ipaleka** [reply]
We have implemented [automatic adding](https://github.com/asastats/channel/issues/128) of Yieldly pools, tho we still need some kind of manual configuration in advance. It looks like Yieldly stabilized around early morning (UTC) Thursday.

**Comment by ipaleka** [comment]
New AlgoFund's ALGF-NURD LP farming program.

**Comment by ipaleka** [comment]
Yieldly pools announced for this week

🦊 YLDY/KITSU LP > YLDY
🦊 YLDY>KITSU

**Comment by ipaleka** [comment]
Some Yieldly pools [are renewed](https://github.com/asastats/channel/issues/165) "immediately" after the old one expires, by creating a new app. Our engine needs an update of such apps too.

**Comment by ipaleka** [comment]
Algofi is going to upgrade their contracts to v2 in a few weeks, so we need to add every new program.

**Comment by ipaleka** [comment]
Yieldly's COSG-COSG is going to be extended this Thursday. https://twitter.com/YieldlyFinance/status/1554421755325820933?s=20&t=2xoMzrXTPmCUs3jxbZj0iA

**Comment by moloch10** [comment]
> Right now we are leaning on the users' request here in our [issues](https://github.com/asastats/channel/issues) or in our community channels after those programs are already deployed on Mainnet.

How do you feel about adding a 'submit a bug/update' kind of form to the site itself which can potentially create tickets? I see on the FAQ, there's a mention about submitting $bugs via this platform but a form makes it even easier.

**Reply by ipaleka** [reply]
We'll add it after mobile app deployment in a form of the Atlassian Jira floating widget and ticket creator.

**Reply by ipaleka** [reply]
It's a javascript widget, where to put it beside the address and home page? /about/ ?

**Comment by wxivd** [comment]
@ipaleka 

Yieldly announced new NFT prize game

https://twitter.com/YieldlyFinance/status/1625471570054418437

Escrow for YLDY staked to join NFT Prize game is
https://algoexplorer.io/address/E3SJQLV7PHEL6J4MX3EESXG7JP5KI35ORAOOX55OLIE2QOK4YMT3UBPA6Y

**Reply by ipaleka** [reply]
Great, tnx, an issue has been created https://github.com/asastats/channel/issues/472. As I understand it after the initial research, we should track users' YLDY while we aren't interested in their tickets and/or in the prize itself.
